packages/geoprocessing/src/toolbox/overlapGroupMetrics.ts

```diff
@@ -130,7 +175,9 @@ export async function overlapGroupMetrics(options: {
   /** The metrics to group */
   metrics: Metric[];
   /** features to overlap, keyed by class ID, use empty array if overlapArea operation */
-  featuresByClass: Record<string, Feature<Polygon>[]>;
+  featuresByClass:
+    | Record<string, Feature<Polygon>[]>
+    | Record<string, Georaster>;
```

Comment:
My understanding is that `overlapGroupMetrics` can now be called with either features or a raster.  That seems to be the main reason to have mixed mixed vector and raster into the same typings?  If so, would you be willing to test that this function can now handle either input?

Reply:
That is the case, yeah. I did keep the name the same and added the extra typing so there wouldn't be a breaking change. It's handling of raster input is tested in `overlapGroupMetrics.test.ts`, and I can add a test of the feature[] input.